spec/unit/inflecto/inflections/uncountable_spec.rb

```diff
@@ -5,16 +5,16 @@
 
   context 'when word given' do
     subject { object.uncountable('water') }
-    its(:uncountables) { should eq(['water']) }
+    its(:uncountables) { should contain_exactly('water') }
```

Comment:
I'd prefer to see `should eql(Set.new(%w[water]))` here for reasons in https://github.com/mbj/inflecto/pull/8/files#r22766596.